### PR TITLE
[Notifications] Added timestamp and object name to copy notifications

### PR DESCRIPTION
**Pull request by juliecwang**
-Added timestamps to all notifications
-Added object name to copy success notification
-Fixes issue #331

### CODE REVIEW

**Comment by juliecwang** [comment]
**Author Checklist**

- Changes address original issue? Y
- Unit tests included and/or updated with changes? Y
- Command line build passes? Y
- Changes have been smoke-tested? Y

**Comment by charlesh88** [comment]
@juliecwang and I worked together on the solution on this, we're good to go.

File: platform/entanglement/src/actions/AbstractComposeAction.js [hunk 1]
```diff
@@ -118,7 +118,6 @@ define(
                 currentParent = this.currentParent,
                 newParent = this.newParent,
                 object = this.object;
-
```
Comment (dtailor90) [inline platform/entanglement/src/actions/AbstractComposeAction.js]: @juliecwang - the following line deletions will make it harder to read (121 and 162). Just revert the changes on this particular file.
